lib/bloc/explorer/api_explorer.dart

```diff
@@ -27,6 +31,7 @@ enum ExplorerQuery {
 class ApiExplorer {
   late ExplorerClient client;
   late Status status;
+  TransactionCache cache = TransactionCache();
```

Comment:
`TransactionCache` is no longer used and we should use the `WalletStorage` from the database.